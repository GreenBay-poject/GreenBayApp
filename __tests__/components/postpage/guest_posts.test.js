import React from 'react';
import renderer from 'react-test-renderer';
import GuestsPosts from '../../../components/postpage/guest_posts';

describe('<GuestsPosts />', () => {
    it('Sample GuestsPosts', () => {
        const tree = renderer.create(<GuestsPosts user={null} />).toJSON();
        console.log(tree)
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('ActivityIndicator');
        // expect(tree.children[0].type).toBe("Text");
        // expect(tree.children[1].type).toBe("TextInput");
        // expect(tree.children[2].type).toBe("TextInput");
        // expect(tree.children[3].type).toBe("View");
        // expect(tree.children[4].type).toBe("Text");


    });
});