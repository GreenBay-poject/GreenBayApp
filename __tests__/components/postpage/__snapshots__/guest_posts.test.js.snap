// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuestsPosts /> Sample GuestsPosts 1`] = `
<ActivityIndicator
  animating={true}
  color="#C00"
  hidesWhenStopped={true}
  size="large"
  style={
    Array [
      Object {
        "height": 80,
      },
    ]
  }
/>
`;
